onnxruntime/core/session/IOBinding.cc

```diff
@@ -8,6 +8,11 @@
 #include "core/framework/utils.h"
 
 namespace onnxruntime {
+IOBinding::~IOBinding() {
+  feeds_.clear();
+  outputs_.clear();
+}
+
```

Comment:
I thought the vectors get destructed and OrtValue destructors called when IOBinding gets destructed even without ".clear()"?

Reply:
they totally do.  i added this code because it helped me catch heap corruption which i didn't know i was corruption the feeds or the outputs.   i don't need the code anymore and will clean i tup.

---
In reply to: [354069305](https://github.com/microsoft/onnxruntime/pull/2534#discussion_r354069305) [](ancestors = 354069305)